### DISCUSSION TITLE
June topics

### DISCUSSION THREAD

**Discussion by adi-shankara** [EarthBitDevs/earthbitdevs#2] [General]
Post topics for June!

**Comment by ubbabeck** [comment]
- “Having a pre-signed transaction that cannot be changed with a fee-rate that is locked in. Relay on being able to get a confirmation within a certain timeframe.” -Murch. [Gloria Zhao and Package Relay - Episode 21](https://podcast.chaincode.com/2022/05/23/gloria-package-relay.html)
- “Mempool is where the ideology we bitcoiners have really translates in to technical problems. So for example this idea of permissionless and censorship resistants. Anyone anywhere should be able to send a payment regardless of what country they live in, their politics and attempted finical censorship. They should be able to just run a raspberry pie bitcoin node and broadcast their transaction on this p2p network where you are able to hide a little bit because all the peers look the same.” -Gloria Zhao.  [Gloria Zhao and Package Relay - Episode 21](https://podcast.chaincode.com/2022/05/23/gloria-package-relay.html)
- [Package Relay Proposal](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-May/020493.html)

**Edit I realised that I misunderstood the setup so will focus on package relay and mempool this session** 

- ~~Sensitive subject, but [What Is BIP 119 And Why Did It Fuel Such Heated Discussions In Bitcoin?](https://bitcoinmagazine.com/technical/what-is-bip-119-bitcoin-controversy-explained) https://github.com/bitcoin/bitcoin/pull/21702~~
- ~~[Silent payments](https://gist.github.com/RubenSomsen/c43b79517e7cb701ebf77eec6dbb46b8)~~

**Comment by adi-shankara** [comment]
https://twitter.com/ajtowns/status/1534284646933880832?s=21&t=g8gDtWOqJqDTuVgQ8Xy7dQ

**Comment by adi-shankara** [comment]
https://gist.github.com/AdamISZ/51349418be08be22aa2b4b469e3be92f

**Comment by adi-shankara** [comment]
https://gist.github.com/RubenSomsen/21c477c90c942acf45f8e8f5c1ad4fae

**Comment by adi-shankara** [comment]
https://twitter.com/roasbeef/status/1534366238507409408

**Comment by adi-shankara** [comment]
https://diyhpl.us/wiki/transcripts/btcpp/2022/fedimint-ecash/

**Comment by adi-shankara** [comment]
https://unchained.com/blog/contributing-bitcoin-core-patience/

**Comment by adi-shankara** [comment]
https://rosenbaum.se/btcphil/#readme

**Comment by vnprc** [comment]
I'd like to present on:
silent payments (already linked above): https://gist.github.com/RubenSomsen/c43b79517e7cb701ebf77eec6dbb46b8
taro: https://docs.lightning.engineering/the-lightning-network/taro
triggering a LN penalty transaction: https://fiatjaf.com/73095980.html
erlay: https://arxiv.org/pdf/1905.10518v2.pdf
what are anchor outputs?: https://fanismichalakis.fr/posts/anchor-outputs/

These are topics I have already prepared to present for my regional meetup. I think this is probably more than enough to fill 1 hour.

**Comment by vnprc** [comment]
Also, this is super cool. Maybe next time. :)

Private Authentication Protocols: https://github.com/sipa/writeups/tree/main/private-authentication-protocols
